Run on 2018-06-05 11:43:22.936416 with command 'java -cp test/bin:test/libs/*:test/libs/a-sweet-library.jar com.root.Main':
		at com.root.Main.main(Main.java:10)

Run on 2018-06-05 11:43:22.936301 with command 'java -cp test/bin:test/libs/*:test/libs/a-sweet-library.jar com.root.Main':
		at com.root.utils.Utils.<init>(Utils.java:8)

Run on 2018-06-05 11:43:22.936186 with command 'java -cp test/bin:test/libs/*:test/libs/a-sweet-library.jar com.root.Main':
		at com.root.Main.<init>(Main.java:28)

Run on 2018-06-05 11:43:22.936013 with command 'java -cp test/bin:test/libs/*:test/libs/a-sweet-library.jar com.root.Main':
	Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 10

Run on 2018-06-05 11:43:22.935826 with command 'java -cp test/bin:test/libs/*:test/libs/a-sweet-library.jar com.root.Main':
	Uh oh, now theres about to be a real exception

Run on 2018-06-05 11:43:22.935654 with command 'java -cp test/bin:test/libs/*:test/libs/a-sweet-library.jar com.root.Main':
	Buuuuuuttt, we're still chugging on

Run on 2018-06-05 11:43:22.935401 with command 'java -cp test/bin:test/libs/*:test/libs/a-sweet-library.jar com.root.Main':
		at com.root.Main.main(Main.java:10)

Run on 2018-06-05 11:43:22.935127 with command 'java -cp test/bin:test/libs/*:test/libs/a-sweet-library.jar com.root.Main':
		at com.root.utils.Utils.<init>(Utils.java:8)

Run on 2018-06-05 11:43:22.934853 with command 'java -cp test/bin:test/libs/*:test/libs/a-sweet-library.jar com.root.Main':
		at com.root.Main.<init>(Main.java:22)

Run on 2018-06-05 11:43:22.934513 with command 'java -cp test/bin:test/libs/*:test/libs/a-sweet-library.jar com.root.Main':
	java.lang.ArrayIndexOutOfBoundsException: 10

Run on 2018-06-05 11:43:22.932625 with command 'java -cp test/bin:test/libs/*:test/libs/a-sweet-library.jar com.root.Main':
	About to throw and exception

Run on 2018-06-05 11:43:22.932247 with command 'java -cp test/bin:test/libs/*:test/libs/a-sweet-library.jar com.root.Main':
	Main constructor, frame should be visible

Run on 2018-06-05 11:43:22.845290 with command 'java -cp test/bin:test/libs/*:test/libs/a-sweet-library.jar com.root.Main':
	Utils constructor

Run on 2018-06-05 11:43:22.845014 with command 'java -cp test/bin:test/libs/*:test/libs/a-sweet-library.jar com.root.Main':
	Main func

Run on 2018-06-05 11:42:24.242967 with command 'java -cp test/bin:test/libs/*:test/libs/a-sweet-library.jar com.root.Main':
	Main func
	Utils constructor
	Main constructor, frame should be visible
	About to throw and exception
	java.lang.ArrayIndexOutOfBoundsException: 10
		at com.root.Main.<init>(Main.java:22)
		at com.root.utils.Utils.<init>(Utils.java:8)
		at com.root.Main.main(Main.java:10)
	Buuuuuuttt, we're still chugging on
	Uh oh, now theres about to be a real exception
	Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 10
		at com.root.Main.<init>(Main.java:28)
		at com.root.utils.Utils.<init>(Utils.java:8)
		at com.root.Main.main(Main.java:10)

Run on 2018-06-05 11:41:32.692386 with command 'java -cp test/bin:test/libs/*:test/libs/a-sweet-library.jar com.root.Main':
	Main func
	Utils constructor
	Main constructor, frame should be visible
	About to throw and exception
	java.lang.ArrayIndexOutOfBoundsException: 10
		at com.root.Main.<init>(Main.java:22)
		at com.root.utils.Utils.<init>(Utils.java:8)
		at com.root.Main.main(Main.java:10)
	Buuuuuuttt, we're still chugging on
	Uh oh, now theres about to be a real exception
	Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 10
		at com.root.Main.<init>(Main.java:28)
		at com.root.utils.Utils.<init>(Utils.java:8)
		at com.root.Main.main(Main.java:10)

Run on 2018-06-05 11:41:15.023265 with command 'java -cp test/bin:test/libs/*:test/libs/a-sweet-library.jar com.root.Main':
	Main func
	Utils constructor
	Main constructor, frame should be visible
	About to throw and exception
	java.lang.ArrayIndexOutOfBoundsException: 10
		at com.root.Main.<init>(Main.java:22)
		at com.root.utils.Utils.<init>(Utils.java:8)
		at com.root.Main.main(Main.java:10)
	Buuuuuuttt, we're still chugging on
	Uh oh, now theres about to be a real exception
	Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 10
		at com.root.Main.<init>(Main.java:28)
		at com.root.utils.Utils.<init>(Utils.java:8)
		at com.root.Main.main(Main.java:10)